plugin.video.regiotv/resources/lib/addon.py

```diff
@@ -30,6 +30,7 @@ def main_menu():
         label = '{name} [COLOR gray]| {label}[/COLOR]'.format(**channel)
         plot = '[B]{name}[/B]\nRegio {label}\n\n[I]{description}[/I]\n\n[COLOR yellow]{website}[/COLOR]'.format(**channel)
         list_item = ListItem(label=label, label2=channel.get('label'), offscreen=True)
+        list_item.setProperty(key='IsInternetStream', value='true' if is_playable else 'false')
```

Comment:
why this change? Afaik this is an internal kodi property. Can't you set the mimetype instead?

Reply:
I cannot remember what I attempted to do here, but it makes no difference.
I have removed it upstream, not sure if you are blocking the release for this?

Reply:
I'd prefer if you remove this yes. O have requested this in the past too.

Reply:
Ok, I will make a new releease 🙄

Reply:
No doubt it originated from the same add-on it was copied from. It has been part of this add-on since its inception.